README.md

```diff
@@ -18,115 +26,317 @@ feedback, please join us there! We look forward to your contribution.
-minions. The policy.cfg specifies which of these files and options are to be
-used for the deployment.
-
-Please refer to the [Policy wiki page](https://github.com/SUSE/DeepSea/wiki/policy)
-for more detailed information.
+You need at least a minimum of 4 nodes to be able to test and use DeepSea properly.
+
+You could set the option `DEV_ENV=true` as an environment variable or pillar variable during the deployment stages to be able to use
+less than 4 nodes. 
+
+### Add DeepSea repo to your admin host
+
+DeepSea needs to be installed on the Salt master node.
+
+On openSUSE distributions, this step could be performed by installing the RPM package [from the openSUSE build service](https://build.opensuse.org/package/show/filesystems:ceph:luminous/deepsea), following the [installation instructions](https://software.opensuse.org//download.html?project=filesystems%3Aceph%3Aluminous&package=deepsea).
+
+On other distributions or when using a development version, install Salt via the
+distribution's preferred installation method and check out the git repository:
+
+```
+$ git clone https://github.com/SUSE/DeepSea.git
+$ cd DeepSea
+$ sudo make install
+```
+
+### Cluster Preparation 
+The cluster deployment process has several phases. First, you need to prepare all nodes of the cluster by configuring Salt and then deploy and configure Ceph.
+The following procedure describes the cluster preparation in detail.
+
+Install a minimum of four machines (or define `DEV_ENV=true` either as an environment variable or a pillar variable during the deployment process) with SUSE Leap 42.3 or Tumbleweed and add the DeepSea repo to your defined "admin" node:
+
+```
+# zypper ar http://download.opensuse.org/repositories/filesystems:/ceph:/luminous/openSUSE_Leap_42.3/filesystems:ceph:luminous.repo
+# zypper refresh
+```
+
+Make sure that each node can resolve the host names of all other nodes. 
+
+The Salt master needs to resolve all its Salt minions by their host names, as well as all Salt minions need to resolve the Salt master by its host name. 
+
+If you don't have a DNS Server you could also add all hosts of your cluster to the ``/etc/hosts``` file. 
+
+Configure, enable, and start the NTP time synchronization server on all nodes:
+
+```
+# systemctl enable ntpd.service
+# systemctl start ntpd.service
+```
+
+Note that the Cluster deployment with DeepSea will not work with Firewall/AppArmor enabled.
+
+Check whether the AppArmor service is running and disable it on each cluster node to avoid problems:
+
+```
+# systemctl disable apparmor.service
+```
+
+Check whether the Firewall service is running and disable it on each cluster node to avoid problems:
+
+```
+# systemctl disable SuSEfirewall2.service
+```
+
+Install DeepSea on the node which will be the Salt master:
+
+```
+# zypper install deepsea
+```
+
+The command installs the `salt-master` and `salt-minion` packages as a dependency as well as Grafana/Prometheus.
```

Comment:
Grafana and Prometheus are not installed as a package dependency. They are installed by salt in stage 2. I'd suggest simply removing the part here.